From ae8cf779998f5950aed1c6d8174ca5965d05ba67 Mon Sep 17 00:00:00 2001
From: Mitali Vagadiya <vmnagajibhai@habana.ai>
Date: Thu, 12 Jun 2025 12:28:45 +0300
Subject: [PATCH] [JSW-705] jgs: module: remove unwanted jgs code

This patch removes unnecessary or additional code
which was kept previously as part of cleanup,
just to pass ci.

Change-Id: I21f053e9b3fe3347c300892f345ee12189100d1c
---

diff --git a/arch/jgs/common.c b/arch/jgs/common.c
index d5fe246..4c295e6 100644
--- a/arch/jgs/common.c
+++ b/arch/jgs/common.c
@@ -113,24 +113,6 @@
 	return 0;
 }
 
-/* REMOVE ONCE CI PASS */
-u32 fs1_get_arc_addr_from_64bit(u64 addr)
-{
-	/* ARC.MNG DCCM and ICCM regions */
-	if (addr >= IO0_MNG_ICCM_FULL_BASE && addr < IO0_MNG_ICCM_FULL_BASE +
-		(u64)IO_0_MNG_ARC_ICCM_SIZE)
-		return (u32)((addr & ADDR_MASK(IO_MNG_ICCM)) + LOCAL_ICCM_BASE);
-
-	if (addr >= IO0_MNG_DCCM_FULL_BASE && addr < IO0_MNG_DCCM_FULL_BASE +
-		(u64)IO_0_MNG_ARC_DCCM_SIZE)
-		return (u32)((addr & ADDR_MASK(IO_MNG_DCCM)) + LOCAL_DCCM_BASE);
-
-	/* TODO Other regions better be added as well (GT, HBW, DBG) */
-
-	/* CFG region (15) */
-	return (addr & 0xffffffff);
-}
-
 u32 jgs_get_arc_addr_from_64bit(u64 addr)
 {
 	/* ARC.MNG DCCM and ICCM regions */
